test/functional/toolchains/microsoft/test_v2_msbuild_template.py

```diff
@@ -49,11 +49,11 @@ def test_msbuild_lib_2022():
     assert "hello/0.1: _MSC_VER191" in client.out
 
     # Create works
-    client.run("create . -s compiler.version=193")
+    client.run("create . -s compiler.version=194")
```

Comment:
I am not sure why this change.
previously neither 193 nor 194 were the default version, yet the test managed to build and pass. Why is it necessary to do this, maybe because 193 toolset is not installed anymore? The problem with this is that we would not test anymore that we can build ``compiler.version=193`` with a VS>=17.10

Reply:
ok, let me try to install the v143 toolset too and check if we can make it work as it should.